kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -65,8 +116,14 @@ To run the generated native executable, generated in `target/`, execute
 
 ### Use the application
 
-To make use of this application it is as simple as putting a message on `travellers` topic with following content (cloud event format) 
+To make use of this application it is as simple as putting a message on `travellers` topic with following content 
+
+* Send Message to Topic
+```
+kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
+```
 
+Content (cloud event format)
 ```
 {
   "specversion": "0.3",
```

Comment:
`type` in this json should be `TravellersMessageDataEvent_3`

Reply:
hi @radtriste are you sure of this type? When I tested it worked, the type you mentioned I think goes in the message sent to the other topic, can you confirm?

Reply:
`VisaApplicationsMessageDataEvent_8` is from a different application ... (travel agency)
It won't show any error, but the user may not understand why it is called like that (check the result `type` field)

Better to put something related to the current application here ;)